client/src/components/TaskLine.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import GlobalStyles from './GlobalStyles';
+import Dot from '../assets/dot.svg';
+import CategoryButton from './CategoryButton';
+
+const Task = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 5px 15px 5px 10px;
+`;
+
+const Time = styled.p`
+  font-size: 15px;
```

Comment:
Better use `rem` instead of `px` for responsive reasons. You use `font-size:15px` three times, maybe its better to define the font-size in the task div. Then you must only define it additionally in the TaskHeading because it's an` h2` tag.

Reply:
that is the plan for the next refactor :)